CONTRIBUTING.md

```diff
@@ -64,14 +64,23 @@ Debugging a Test Case
 To debug a test case, we can simply run docker container as mentioned above by keeping the shell connected (change mentioned in step no 5).
 
 1. Switch to monitoring agent directory in Docker container:
-```
-cd /agent2/src/rackspace-monitoring-agent
-```
-2. Run Test to verify:
-```
-./luvi-sigar . -m tests/run.lua 
-```
+    ```
+    cd /agent2/src/rackspace-monitoring-agent
+    ```
```

Comment:
Markdown code blocks are either indented four spaces or surrounded by the triple-backtick. You don't need to do both. For example:

```
This is a backticked code block.
```

    This is an indented code block.

See?